aws-lambda-java-runtime-interface-client/src/main/jni/deps/aws-lambda-cpp-0.2.7/include/aws/lambda-runtime/runtime.h

```diff
@@ -61,6 +61,11 @@ struct invocation_request {
      */
     std::chrono::time_point<std::chrono::system_clock> deadline;
 
+    /**
```

Comment:
hopefully at some point, we won't have a copy of aws-lambda-cpp in here, so let's keep that change here but I think you also need to modify the source of truth here: https://github.com/awslabs/aws-lambda-cpp?

Reply:
Agreed, one day, we could get rid of this. (Not blocking PR from my end either).